Changelog.md

```diff
@@ -16,7 +16,8 @@ Bugfixes:
  * Code Generator: Fix ICE when doing an explicit conversion from ``string calldata`` to ``bytes``.
  * Control Flow Graph: Perform proper virtual lookup for modifiers for uninitialized variable and unreachable code analysis.
  * Immutables: Fix wrong error when the constructor of a base contract uses ``return`` and the parent contract contains immutable variables.
- * IR Generator: Fix IR syntax error when copying storage arrays of structs containing functions.
+  * IR Generator: Fix IR syntax error when copying storage arrays of structs containing functions.
```

Comment:
Weird... why is this part of the diff? Looks identical to me.

Reply:
Ah sorry. An extra space by accident. Fixed.